cloud-init/write-archive-artifact-data-to-bucket.tpl.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# This is a Terraform template file, and the input variables are
+# passed in via templatefile().
+#
+# shellcheck disable=SC2154
```

Comment:
Do we need a reason here?

Reply:
We could probably explain this a bit more verbosely, but we use the same text in all of our `*.tpl.sh` files in this directory:
```
# This is a Terraform template file, and the input variables are
# passed in via templatefile().
```

Reply:
I think this is fine as it stands, although if @felddy wants to suggest new language I will happily consider it.  We already adequately explain the use of `shellcheck disable`.